pom.xml

```diff
@@ -285,6 +278,24 @@
             <scope>test</scope>
         </dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-api</artifactId>
+			<version>${legacyuiVersion}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-omod</artifactId>
+			<version>${legacyuiVersion}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
No omod dependencies should be in the parent pom or api pom, these should only be in the omod pom or distro pom.

Reply:
I have moved legacyui-omod dependency to the omod pom.